tests/MissingPhraseHttpException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Tests\Tnapf\Router;
+
+use Tnapf\Router\Exceptions\HttpException;
+
+class MissingPhraseHttpException extends HttpException
+{
+    public const CODE = 0;
+    public const PHRASE = "";
+    public const DESCRIPTION = "Indicates that the server encountered an unexpected
+condition that prevented it from fulfilling the request.";
```

Comment:
why in the world is there a new line here

Reply:
PSR complains about the line being too long. I think I find a better way to get around it though.

Reply:
see my other comment on this line